ruleset.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<ruleset name="Centreon Standard">
+  <description>Custom rules defined by Centreon</description>
+
+  <exclude-pattern>centreon-build</exclude-pattern>
+  <exclude-pattern>GPL_LIB</exclude-pattern>
+  <exclude-pattern>tests</exclude-pattern>
+  <exclude-pattern>vendor</exclude-pattern>
+  <exclude-pattern>*\.(?!(php$))</exclude-pattern>
+
+  <rule ref="PSR2"/>
+  <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace">
+    <properties>
+      <property name="ignoreBlankLines" value="false"/>
+    </properties>
+  </rule>
+
+  <rule ref="PSR1.Files.SideEffects.FoundWithSymbols">
```

Comment:
IMHO even if we do not plan on fixing the _FoundWithSymbols_ and _MissingNamespace_ warnings, they should not be discarded.

Reply:
For now, we won't display these one. Sooner, we'll do it ;)
Wait for the PSR12 update and APIv2 introduction

Reply:
So please display these warnings so that they disappear once fixed :-) You know just like me that _soon_ is a rather relative notion.

Reply:
we know that we can't fix this until the legacy is refacto. We'll only add comments on unfixable inspections... Won't it be useless for now ?
you know that, as soon as I can, I'll add them myself :)))
let it be ignored for now